src/index.js

```diff
@@ -210,11 +214,11 @@ const handleFooterElementOnClick = (tabName) => {
 
 const removeElementsOnTabChange = (tabName) => {
     const allTabIds = [
```

Comment:
No.

Go to any website where you can run js line mdn and solve this problem

// Having object zeta
const zeta = {
alfaTab: null,
betaTab: null,
ceta: null,
delta: null
}

// Create variable beta to with value of ['alfaTab', 'betaTab']
// Of course don't do it manualy, but transform zeta.